scripting/internal/js/iterator.go

```
@@ -9,55 +9,76 @@ import (
+		for v := range s {
+			if !yield(idx, v) {
+				return
+			}
+			idx++
+		}
+	}
+}
+
+// entry encodes a single "entry" in "entries". For the ValueIterator, the
+// "key", i.e. index, is ignored.
+func (i ValueIterator[T, U]) entry(scope Scope[U], _ index, v T) (Value[U], error) {
```

golangci-lint (2.12.2)

[error] func ValueIterator[T, U].entry is unused

(unused)